From 197dc1fd3cfe3d4b22280be241bfe1a85b8d37e6 Mon Sep 17 00:00:00 2001
From: Chia-I Wu <olv@google.com>
Date: Tue, 3 Jul 2018 01:17:18 +0800
Subject: [PATCH 11/48] surfaceflinger: default to
 DisplayColorSetting::ENHANCED

DisplayColorSetting works like a hint.  On devices that do not
support RenderIntent::ENHANCE, RenderIntent::COLORIMETRIC will be
picked.

Bug: 79434305
Test: Pixel 2017 and 2018 boot animations
Change-Id: Ie85cca6eae75cf6ada3a4a364c402d09dc2920f9
---
 services/surfaceflinger/SurfaceFlinger.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/services/surfaceflinger/SurfaceFlinger.h b/services/surfaceflinger/SurfaceFlinger.h
index 87bc9ccb0..afcefdb94 100644
--- a/services/surfaceflinger/SurfaceFlinger.h
+++ b/services/surfaceflinger/SurfaceFlinger.h
@@ -884,7 +884,7 @@ private:
     static bool useVrFlinger;
     std::thread::id mMainThreadId;
 
-    DisplayColorSetting mDisplayColorSetting = DisplayColorSetting::MANAGED;
+    DisplayColorSetting mDisplayColorSetting = DisplayColorSetting::ENHANCED;
     // Applied on Display P3 layers when the render intent is non-colorimetric.
     mat4 mEnhancedSaturationMatrix;
 
-- 
2.17.1

